spec/factories/assessments.rb

```diff
@@ -75,6 +75,12 @@
       end
     end
 
+    trait :with_unrequested_professional_standing_request do
```

Comment:
```suggestion
    trait :with_received_professional_standing_request do
```

To match the states we've got already.